apps/src/imageUtils.js

```diff
@@ -113,8 +91,56 @@ export function canvasToBlob(canvas) {
   });
 }
 
-export function dataURIToBlob(uri) {
-  return imageFromURI(uri)
-    .then(canvasFromImage)
-    .then(canvasToBlob);
+export async function dataURIToBlob(uri) {
+  const canvas = await toCanvas(uri);
+  return await canvasToBlob(canvas);
+}
+
+/**
+ * Given an input of a supported type, converts it to an HTMLCanvasElement.
+ *
+ * @param {string|HTMLImageElement|HTMLCanvasElement} input
```

Comment:
Good point.  To avoid writing the same doc here and for the method below I've defined `ImageURI` with a JSDoc `@typedef` and used it in the param signatures both places.